tests/sentry/api/endpoints/test_team_projects.py

```diff
@@ -50,6 +50,9 @@ def test_simple(self):
 
 @region_silo_test(stable=True)
 class TeamProjectsCreateTest(APITestCase):
+    endpoint = "sentry-api-0-team-project-index"
+    method = "post"
```

Comment:
I am breaking up my huge PR https://github.com/getsentry/sentry/pull/55010. As part of that change, I refactor this page to use `self.get_success/error_response` so that's why these are here now

Reply:
Thanks for breaking it up. They're reasonably packaged.